src/components/pages/Landing/Landing.test.tsx

```diff
@@ -348,7 +348,7 @@ describe('Landing Component', () => {
         await waitFor(() => {
             expect(
                 screen.getByText(
-                    'Market WFIL-DEC2024 is open for pre-orders now until May 31, 2023 23:00 (UTC)'
+                    'Market WFIL-DEC2024 is open for pre-orders now until May 31, 2023 (UTC)'
```

Comment:
not expecting this text to change. It was recently fixed to show 23:00

Reply:
Fixed this. Was mistaken on which change to accept from git merge